dali/python/nvidia/dali/auto_aug/auto_augment.py

```diff
@@ -45,8 +45,8 @@ def auto_augment_image_net(sample: _DataNode, shape: Optional[_DataNode] = None,
     ---------
     sample : DataNode
         A batch of samples to be processed. The samples should be images of `HWC` layout,
-        `uint8` type and reside on GPU.
-    shapes: DataNode, optional
+        `uint8` type.
+    shape: DataNode, optional
```

Comment:
On the one hand, it might be a bit controversial, on the other, it just fits the "sample" oriented style of DALI pipelines (maybe apart from manual/external source feeding of the inputs). You write the pipeline as if in terms of samples, but it produces batches. I am also reluctant to chaning the parameter name, as we already released this module, it would be kind of a breaking change.

Reply:
But you _did_ change the other - I know it's optional, but isn't the first argument positional anyway?

Reply:
The change is only in the docs - I've mistekenly used plural in the docs, the actual argument name is shape.

Reply:
Btw. I apparently fixed that typo in https://github.com/NVIDIA/DALI/pull/4753, after rebasing it dissapeared here.